src/SubscribeStar/README.md

```diff
@@ -0,0 +1,49 @@
+# Patreon
```

Comment:
```suggestion
# Subscribestar
```

Reply:
oof How'd I miss that one search and replace failed meee :)